shared/src/helpers/user/get-avatar-initials.ts

```diff
@@ -1,9 +1,12 @@
 const getAvatarInitials = (fullName: string): string => {
     const [name, secondName] = fullName.split(' ');
     const firstCharIndex = 0;
-    return `${name[firstCharIndex].toUpperCase()}${secondName[
-        firstCharIndex
-    ].toUpperCase()}`;
+
+    return secondName
+        ? `${name[firstCharIndex].toUpperCase()}${secondName[
```

Comment:
we should add logic for email. Because after signup we dont have name

Reply:
we don't use t anymore since new Avatar, it's old helper for old avatar, so I can only delete it.
![Screenshot 2023-09-11 232810](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/24193459/197cbc4b-2b63-4f6c-9120-49311f0a2b67)